src/main/java/org/wildfly/security/audit/SyslogAuditEndpoint.java

```diff
@@ -33,7 +33,7 @@
 import javax.net.SocketFactory;
 
 /**
- * An {@link AuditEndpoint} that logs to syslog.
+ * An audit endpoint that logs to syslog server.
```

Comment:
Not going to hold up the PR for this.

For changes like this is there a reason to remove all links?  These allow automatic cross referencing in the generated javadoc.

Reply:
This links only duplicates links generated from "implements" of class. So based on recommendation:
> Only use @link on the first reference to a specific class or method. Use @code for subsequent references. This avoids excessive hyperlinks cluttering up the Javadoc.